src/core/Akka/Actor/ActorSelection.cs

```diff
@@ -57,24 +59,26 @@ public ActorSelection(IActorRef anchor, SelectionPathElement[] path)
         }
 
         /// <summary>
-        ///     Initializes a new instance of the <see cref="ActorSelection" /> class.
+        /// Initializes a new instance of the <see cref="ActorSelection" /> class.
         /// </summary>
         /// <param name="anchor">The anchor.</param>
         /// <param name="path">The path.</param>
         public ActorSelection(IActorRef anchor, string path)
             : this(anchor, path == "" ? new string[] {} : path.Split('/'))
         {
         }
-
+        
         /// <summary>
-        ///     Initializes a new instance of the <see cref="ActorSelection" /> class.
+        /// Initializes a new instance of the <see cref="ActorSelection" /> class.
         /// </summary>
         /// <param name="anchor">The anchor.</param>
         /// <param name="elements">The elements.</param>
         public ActorSelection(IActorRef anchor, IEnumerable<string> elements)
         {
             Anchor = anchor;
+            
             Path = elements
+                .Where(s=>!string.IsNullOrWhiteSpace(s))
```

Comment:
This handles empty elements, for example a trailing /, otherwise it would result in a additional `SelectChildName`